docs/Docker-Support.md

```diff
@@ -79,6 +79,25 @@ ssh:
   auth_methods: ["password"]
 ```
 
+### Mounting volumes into your docker container ###
+You can mount folders into a docker container:
+
+    HOSTS:
+      ubuntu-12-10:
+        platform: ubuntu-12.10-x64
+        image: ubuntu:12.10
+        hypervisor: docker
+        mount_folders:
+          name1:
+            from: host_path1
```

Comment:
from/to was taken from the ticket.  I assume that's because that's what they're named for vagrant.

I prefer your more explicit names, and have changed them